nf_core/modules/lint/module_tests.py

```diff
@@ -19,69 +19,85 @@ def module_tests(_, module):
     file.
 
     """
-
+    nftest_testdir = os.path.join(module.component_dir, "tests")
+    if os.path.exists(nftest_testdir):
+        module.passed.append(("test_dir_exists", "nf-test test directory exists", nftest_testdir))
     if os.path.exists(module.test_dir):
         module.passed.append(("test_dir_exists", "Test directory exists", module.test_dir))
     else:
         module.failed.append(("test_dir_exists", "Test directory is missing", module.test_dir))
+        module.warned.append(("test_dir_exists", "nf-test directory is missing", nftest_testdir))
```

Comment:
The nf-test warning should be moved to the else after checking if nf-test dir exists.
Or use if/elif/else otherwise

Reply:
That was a typo :grin: The second if statement should have been elif :p